src/index.njk

```diff
@@ -2,4 +2,8 @@
 title: Index
 ---
 
+PNG+GIF
 {% photoFolder "src/assets/images" %}
```

Comment:
By default the `photoFolder` shortcode only returns *.png and *.gif files (but not for any good reason, only for example purposes).